### PR TITLE
dev-libs/openssl: bump to 1.0.2g

### PR DESCRIPTION
**Pull request by mischief**
https://bugs.gentoo.org/show_bug.cgi?id=575548

### CODE REVIEW

File: profiles/coreos/base/package.accept_keywords [hunk 1]
```diff
@@ -80,6 +80,10 @@ dev-util/checkbashisms
 # https://github.com/golang/go/issues?q=milestone%3AGo1.4.3
 =dev-lang/go-1.4.3 ~amd64
 
+# 1.0.2g contains some security fixes.
+# https://bugs.gentoo.org/show_bug.cgi?id=575548
+=dev-libs/openssl-1.0.2g ~amd64 ~arm64
```
Comment (marineam) [inline profiles/coreos/base/package.accept_keywords]: Edit the keywords in the ebuild, this is just extra tediousness for overlay ebuilds.

Comment (mischief) [inline profiles/coreos/base/package.accept_keywords]: duh, woops. they are already not ~ in the ebuild :)

Comment (marineam) [inline profiles/coreos/base/package.accept_keywords]: Actually, the ebuild doesn't have other changes so keywords are correct and this is doing nothing.

Comment (marineam) [inline profiles/coreos/base/package.accept_keywords]: Yup

**Comment by marineam** [comment]
LGTM

**Pull request merged**